docs/readmes/architecture/agw_overview.md

```diff
@@ -0,0 +1,59 @@
+---
+id: agw_overview
+title: AGW Overview
+hide_title: true
+---
+
+# Access Gateway Overview
+
+The main service within the access gateway (AGW) is Magmad, which brings up all the services hosted within the AGW. The other major services and components hosted within the AGW are: the MME, S-PGW, Health Checker, Mobilityd, Sessiond/PCEF, Pipelined, PolicyDB, Subscriberdb, OVS-data path, Enodebd, and the Control Proxy. Together, these components help to facilitate and manage data both to and from the user. 
```

Comment:
In stateless mode, sctpd is not tied to Magmad service. I also noticed it is not listed among the major services?

Reply:
I will mention it by updating the first sentence to "The main service within the access gateway (AGW) is Magmad, which brings up all the services hosted within the AGW with the exception of the sctpd service when in stateless mode."